Data Model For Job Entity.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE "Job" (
+  "job_id" INTEGER PRIMARY KEY,
+  "title" VARCHAR NOT NULL,
+  "level_id" INTEGER NOT NULL,
+  "category_id" INTEGER NOT NULL,
+  "location" VARCHAR,
+  "description" TEXT,
+  "compensation" VARCHAR,
+  "application_link" VARCHAR,
+  "is_external" BOOLEAN DEFAULT false,
+  "company_id" INTEGER,
+  "created_at" TIMESTAMP DEFAULT (now()),
+  "updated_at" TIMESTAMP DEFAULT (now())
+);
+
+CREATE TABLE "Job_Category" (
+  "category_id" INTEGER PRIMARY KEY,
```

Comment:
i would keep the naming simple here. We already know the table name is Job_Category, we can name the columns id and title